packages/terra-filter-view/src/_Frame.jsx

```diff
@@ -0,0 +1,540 @@
+      // Close the dropdown
+      this.setState({
+        isOpen: false,
+      });
+    }
+
+    if (this.props.onSelect) {
+      this.props.onSelect(value, option);
+    }
+  }
+
+  /**
+   * Toggles the dropdown open or closed.
+   */
+  toggleDropdown() {
+    // Trigger search callback on press
+    this.delaySearch();
+    if (this.props.variant === Variants.LIST) {
+      return;
+    }
+    if (this.state.isOpen) {
+      this.closeDropdown();
+    } else {
+      this.openDropdown();
+    }
+  }
+
+  render() {
+    const {
+      disabled,
+      disableAutoSearch,
+      display,
+      dropdown,
+      dropdownAttrs,
+      isInvalid,
+      minimumSearchTextLength,
+      noResultContent,
+      onDeselect,
+      onInvalidSearch,
+      onSearch,
+      onSelect,
+      optionFilter,
+      placeholder,
+      searchDelay,
+      showResultsInitially,
+      variant,
+      value,
+      ...customProps
+    } = this.props;
+
+    const selectClasses = cx([
+      'select',
+      variant,
+      { 'is-above': this.state.isAbove },
+      { 'is-focused': this.state.isFocused },
+      { 'is-open': (this.state.isOpen && variant === Variants.DROPDOWN) || (variant === Variants.LIST && this.state.searchValue) },
+      customProps.className,
+    ]);
+
+    let ariaOwns;
+    if (this.state.isOpen && variant === Variants.DROPDOWN) {
+      ariaOwns = 'terra-filter-dropdown';
+    } else if (variant === Variants.LIST) {
+      ariaOwns = 'terra-filter-list';
+    }
+
+    return (
+      <div
+        className={cx(['select-parent', { 'is-disabled': disabled }])}
+        onMouseDown={this.handleMouseDown}
+        onBlur={this.handleBlur}
+      >
+        <div
+          {...customProps}
+          role="combobox"
+          aria-controls={this.state.isOpen ? ariaOwns : undefined}
+          aria-disabled={!!disabled}
+          aria-expanded={!!this.state.isOpen}
+          aria-haspopup={variant === Variants.DROPDOWN ? 'true' : undefined}
+          aria-owns={this.state.isOpen ? ariaOwns : undefined}
+          className={selectClasses}
+          onKeyDown={this.handleKeyDown}
+          ref={(select) => { this.select = select; }}
+        >
+          {/* eslint-disable-next-line jsx-a11y/interactive-supports-focus */}
+          <div
+            className={cx(['display', { 'is-disabled': disabled }])}
+            onMouseDown={this.openDropdown}
+            onFocus={this.handleFocus}
+            role="textbox"
+          >
+            {this.getDisplay()}
+          </div>
+          {/* Render a clear search button when text is present in input */}
+          {this.state.searchValue && this.state.isFocused &&
+            <IconIncomplete
+              className={cx('clear-button')}
+              onClick={this.clearSearch}
+              onMouseEnter={this.handleMouseEnter}
+              onMouseLeave={this.handleMouseLeave}
+            />
+          }
+          <Button
+            className={cx(['button'])}
```

Comment:
nit: array brackets are unneeded. 

Brackets are actually unneeded for cx entirely. Cx will accept any list of arguments and couple them together. `cx('one', 'two', 'three')` is equivalent to `cx(['one', 'two', 'three'])`